src/data.tsx

```diff
@@ -0,0 +1,94 @@
+      "definition": "Your down payment is the amount of money you put down on your mortgage. Your down payment is due during closing and is usually the most expensive closing cost you need to plan for.",
+      "information": "You might have heard you need 20% down to buy a home. The reason why this number is often quoted is that 20% down is the minimum you’ll need to avoid buying private mortgage insurance – it’s not the minimum you need to get a loan.",
+      "note":"You can buy a home with as little as 3% down.",
+      "source": "https://www.rocketmortgage.com"
+    },
+    "mortgage_term": {
+      "type": "Mortgage Term", 
+      "question": "What mortgage term are you aiming for?",
+      "description": "Your mortgage term is the number of years you’ll pay on your loan before you fully own your home. ",
+      "information": "A mortgage is a type of loan that uses your home as collateral. It’s typically used to buy a home or in refinance situations to secure a more favorable deal and possibly convert equity in your home to cash.",
+      "note": "Typically, a fixed-rate mortgage runs 30 years and this is the number we will use if you want us to.",
+      "source": "https://www.rocketmortgage.com/"
+    },
+    "rent": {
+      "type": "Rent", 
+      "question": "IMAGINATIVE: What is your current monthly rent?",
+      "description": "",
+      "information": "",
+      "note": "The imaginative report gives a total home price you can afford based off of your monthly rent.",
+      "source": ""
+    },
+    "goal_home_price": {
+      "type": "Goal Home Price", 
+      "question": "PRAGMATIC: What is the price of your ideal home?",
+      "description": "",
+      "information": "",
+      "note": "The pragmatic report gives your monthly payment you would make with your goal price.",
+      "source": ""
+    },
+    "external_services": {
+      "type": "",
+      "property_tax": "number",
+      "description": "",
+      "information": "",
+      "note":"",
+      "affordability": "mortgage amount + average mortgage from zipcode"
+    },
+    "survey":{
+      "type": "survey",
+      "question": "",
+      "description": "While every person’s situation is different we aim to provide you with a report that would give you a better perspective on where you are on your dream home quest. We based our data on your net monthly income (that’s after taxes), monthly debt, income, downpayment and more",
+      "information": "One of the main questions to answers is how much can I afford?. Affordability is defined as the cost of something.",
+      "note": "Here are the generally recommended guidelines: * Your mortgage payment should be 28% or less. * Your debt-to-income ratio (DTI) should be 36% or less. * Your housing expenses should be 29% or less. This is for things like insurance, taxes, maintenance, and repairs. * You should have three months of housing payments and expenses saved up.",
+      "source": "http://www.trulia.com"
+    }
+  }
```

Comment:
I think this covers all the data needed.

Reply:
Heads up:

Backend discovered some CRUD functionality that we think would be useful.

Say, in the future, you want to update the source link and description for once of these items.  When we add this functionality, you'll be able to post, update, delete, on top of get.

It's currently outside of MVP, but it's coming.